tsl/test/sql/chunk_api.sql.in

```diff
@@ -398,6 +398,18 @@ SELECT tablespace FROM pg_tables WHERE tablename = :'CHUNK_NAME';
 
 DROP TABLE chunkapi;
 
+-- Test creating a chunk from an existing chunk table
+CREATE TABLE chunkapi (time timestamptz not null, device int, temp float);
+SELECT * FROM create_hypertable('chunkapi', 'time');
+
+CREATE TABLE newchunk (LIKE chunkapi);
```

Comment:
Can we add a test case where the rename of the provided chunk_table is not needed?

Reply:
There should be test cases for both same-named chunk and different-named chunk table now.